modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -121,22 +144,20 @@ $ ip vrf show
 ----
 +
 .Example output
-+
 [source,terminal]
 ----
 Name              Table
 -----------------------
-red                 10
+vrf-1             1001
 ----
-. Confirm the VRF interface is master of the secondary interface:
+. Confirm that the VRF interface is master of the additional interface:
```

Comment:
This needs to be rephrased to either remove the term "master" or (if that is not possible), reframe it as a part of the code. For machines, we sometimes must refer to the control plane nodes as "nodes that have the `master` role", since `role: master` is in the spec. I don't know what the equivalent is for this case, but (if you can't omit the term entirely) find out how to make it describe its technical aspect specifically.